app/assets/javascripts/models/scorecard.js

```diff
@@ -6,6 +6,7 @@ App.Scorecard = DS.Model.extend({
   user: DS.belongsTo('user', { async: true }),
   userInitials: DS.attr('string'),
   turns: DS.hasMany('turn', { async: true }),
+  isStarted: DS.attr('boolean'),
```

Comment:
Yeah, that does seem unfortunate. It's not a simple validation, so maybe it's inevitable.